library/datasort.c

```diff
@@ -94,13 +94,14 @@ static int datasort_base_get_path(struct eblob_backend *b, struct eblob_base_ctl
 	return 0;
 }
 
-static int datasort_chunk_get_path(struct eblob_backend *b, struct eblob_base_ctl *bctl,
+static int datasort_chunk_get_path(struct datasort_ctl *dctl, struct eblob_base_ctl *bctl,
```

Comment:
it's quite hard to distinguish bctl and dctl during reading, they look practically the same

Reply:
Which name do you think is more appropriate?

Reply:
Renamed to `ds_ctl`